README.md

```diff
@@ -2,25 +2,41 @@
 
 ## Pre-requisites
 
-- Node - Instructions to install can be found here: https://nodejs.dev/learn/how-to-install-nodejs (This was tested with version 12.19.0)
+- Node - Instructions to install can be found here: _[How to install `nodejs`](https://nodejs.dev/learn/how-to-install-nodejs)_ 
+- This project was tested with [`nodejs` version 12.19.0](https://nodejs.org/en/download/releases/)
+- ***(Optional)*** Use `nvm` to manage `nodejs` versions: _[How to install and manage `nvm`](https://github.com/nvm-sh/nvm#installing-and-updating)_
+
+## To run the apps
+
 - Clone this repo to your local computer
+- ***(Optional)*** Navigate to the main folder in a terminal and use command `nvm use` to set the same `nodejs` version 
 - Navigate to the main folder in a terminal and use command `npm i` to install needed packages
+- Navigate to the main folder and type command `npm run start:frontend` to start the front end. Navigate to `http://localhost:4200/`
+- Navigate to the main folder and type command `npm run start:backend` to start the back end. Navigate to `http://localhost:3333/songs` for the base path
 
-## To run the apps
+```shell
+git clone https://github.com/atbriggs/nsync
+cd nsync 
+nvm use
+npm i 
+npm run start:frontend
```

Comment:
I think I prefer specifying that these are in different folders
```
Running the Frontend
- Navigate to the frontend folder
- `npm run start:frontend` to start the front end
- Server will be listening on http://localhost:4200/

Running the Backend
- Navigate to the backend folder
- `npm run start:backend` to start the back end
- Navigate to http://localhost:3333/songs for the base path 
```

Reply:
or `cd frontend` / `cd backend` ... likely easier for students who want to get it installed and running ASAP lol

Reply:
That is a great improvement. On the second review, don't you think `cd /frontend` would be misleading, given that both `npm run start:frontend` and `npm run start:backend` commands work when executed from the project root directory?

Reply:
LOL good point @murindwaz! please disregard. that's how I interpreted it and that's how I've been running it 😅  but good point